page-home.php

```diff
@@ -0,0 +1,123 @@
+
+<?php
+/*
+ * Template Name: Currency Converter
+ *
+ */
+get_header(); ?>
+
+<?php get_template_part( 'breadcrumb' ); ?>
+
+<div id="primary" class="content-area level-one">
+    <div class="container">
+        <div class="row" role="banner">
+            <div class="col-md-12">
+                <?php
+                global $post;
+                $image      = make_path_relative_no_pre_path( get_feature_image_url( $post->ID, 'full-page-width', true ) );
+                $title      = get_the_title();
+                get_page_banner( 'level one', $title, $image, '', '', '' );
+                ?>
+            </div>
+        </div>
+
+            <div class="row">
+                <main role="main">
+            <div class="col-md-9">
+                <div class="row">
+                    <div class="col-md-12">
+                        <article role="application">
+                            <div class="entry-header">
+                                <h2>Currency Converter</h2>
+                            </div>
+                            <div class="row entry-content">
+                                <div class="col-md-12">
+                                    <form action="index.php" method="post" id="currency-form" role="application">
```

Comment:
Please remove `role="application"` as it's not needed.

Reply:
Hi @punalsc , If I am remembering correctly @gtvj recommended I keep that in for accessibility.

Reply:
@AshTNA, again i think this is technically correct, but i would suggest testing it with a mac screen reader, so that there is no interference with normal web navigation controls, and the users can tab through the fields.

`application: A region declared as a web application, as opposed to a web document. (note: The role of application should only be used with caution because it gives a signal to screen reading software to turn off normal web navigation controls. Simple widgets should generally not be given the application role, nor should an entire web page be given the application role, unless it is not to be used at all like a web page, and not without much user testing with assistive technology.)`

https://www.w3.org/WAI/GL/wiki/Using_ARIA_landmarks_to_identify_regions_of_a_page

Reply:
@punalsc - I agree. Can you create a PR on the development guide which provides guidance on the need to exercise caution when using `role=application`?

Reply:
Hi @punalsc , I do not have a Mac, would this be OK? https://www.nvaccess.org/